_templates/hello.html

```diff
@@ -5,7 +5,7 @@
 
 <aside class="bio-info">
   <div class="name">
-      {{ }}
+      {{ name }}
```

Comment:
@enmanuelxmorillo this syntax `{{ var_name }}` looks for key in the `html_context` variable, since you had empty `{{ }} ` it was looking for a key of ` ` in the variable and that caused the issue. Putting this back fixes it.

Reply:
cc: @Obarrie here is the fix